lifecycle-operator/webhooks/pod_mutator/pod_mutating_webhook_test.go

```diff
@@ -1207,6 +1208,125 @@ func TestPodMutatingWebhook_Handle_SingleService(t *testing.T) {
 	}, workload.Spec)
 }
 
+func TestPodMutatingWebhook_Handle_SchedulingGates(t *testing.T) {
```

Comment:
let's also add a test case for a pod where the gates have been removed previously